jupyter/datascience/ubi8-python-3.8/requirements.in

```diff
@@ -0,0 +1,18 @@
+# JupyterLab data science requirements
+scikit-learn==0.24.1
+kafka-python==2.0.2
+matplotlib==3.4.1
+pandas==1.2.4
+numpy==1.19.2
+boto3==1.17.11
+scipy==1.6.2
+
+# JupyterLab extensions
+jupyter-resource-usage==0.6.1
+jupyterlab-s3-browser==0.10.1
+jupyterlab-widgets==3.0.3
+jupyterlab-lsp==3.10.0
+jupyterlab-git==0.30.0
+jupyter-bokeh==3.0.4
+plotly==5.5.0
+nbdime==3.0.0
```

Comment:
`jupyterlab-git` and `nbdime` are already included in the minimal notebook image, no need to include them twice